components/builder-http-gateway/src/http/helpers.rs

```diff
@@ -359,7 +360,7 @@ pub fn promote_job_group_to_channel(
     {
         return Err(NetError::new(
             ErrCode::GROUP_NOT_COMPLETE,
-            "hg:promote-job-group:0",
+            "hg:promote-or-demote-job-group:0",
```

Comment:
I'm not really sure what these strings are used for. Is this ok?

Reply:
These strings are meant to be a globally unique identifier associated with each error, so if we see one in e.g. a log, you can grep the codebase and find the one place where that occurred.  They're just for making tracking errors down easier, and they tend to be named after the module/function they're located in.  The change you've made is fine.